GameMod/MPAutoSelection.cs

```diff
@@ -1036,7 +1036,7 @@ public static bool Prefix(Player __instance)
                         }
                     }
                 }
-                return true;

```

Comment:
This change makes it so `MaybeSwitchToNextMissile` does nothing if opt_primary_autoswitch is not 0 or secondarySwapFlag is false.  I don't think that's what you want.  `return true;` needs to happen when autoselect is disabled for missiles.